pandas/tests/test_format.py

```diff
@@ -2343,7 +2343,21 @@ def test_csv_to_string(self):
         df = DataFrame({'col' : [1,2]})
         expected = ',col\n0,1\n1,2\n'
         self.assertEqual(df.to_csv(), expected)
-
+        
+    def test_to_csv_decimal(self):
+        df = DataFrame({'col1' : [1], 'col2' : ['a'], 'col3' : [10.1] })
```

Comment:
I've tried to squash the commits as described but I faced the following error:
$git rebase pandas/master
fatal: Needed a single revision
invalid upstream pandas/master

Any idea ?